libs/execution_context/ebpf_core.c

```diff
@@ -2475,17 +2503,11 @@ ALIAS_TYPES(get_handle_by_id, get_map_handle_by_id)
 ALIAS_TYPES(get_handle_by_id, get_program_handle_by_id)
 
 static ebpf_protocol_handler_t _ebpf_protocol_handlers[] = {
-#if !defined(CONFIG_BPF_JIT_DISABLED)
     DECLARE_PROTOCOL_HANDLER_VARIABLE_REQUEST_VARIABLE_REPLY(resolve_helper, helper_id, address, PROTOCOL_JIT_MODE),
     DECLARE_PROTOCOL_HANDLER_VARIABLE_REQUEST_VARIABLE_REPLY(resolve_map, map_handle, address, PROTOCOL_JIT_MODE),
-#endif
```

Comment:
I don't think we need the actual prototypes or structs do we? 
It seems like just empty entries in the table should be sufficient to fix the IOCTL numbers.
And then the place where they're accessed should just check for a null function pointer.